pkg/util/util.go

```diff
@@ -33,3 +36,65 @@ func GetRequestHost(req *http.Request) string {
 	}
 	return host
 }
+
+// RefreshValue contains a composite cookie refresh value setting
+type RefreshValue struct {
+	RefreshDuration *time.Duration
+	RefreshPercent  uint8
+}
+
+// Set a value of a refresh
+func (r *RefreshValue) Set(s string) error {
+	origS := s
+	s = strings.TrimSpace(s)
+	if strings.HasSuffix(s, "%") {
+		i, err := strconv.ParseInt(s[:len(s)-1], 10, 8)
```

Comment:
The one downside to squeezing this into a `uint8` is it results a more obscure error message when a % that is too large is given where the next error handler would print a better message:

```
percentage "10000" must be a valid integer: strconv.ParseInt: parsing "10000": value out of rangeint: 127
```

Reply:
I'll step over this with an error message that skips the parser error entirely.